public/app/core/utils/fetch.ts

```diff
@@ -74,7 +74,10 @@ export const isContentTypeApplicationJson = (headers: Headers) => {
   }
 
   const contentType = headers.get('content-type');
-  if (contentType && contentType.toLowerCase() === 'application/json') {
+  if (
+    contentType &&
+    (contentType.toLowerCase() === 'application/json' || contentType.toLowerCase() === 'application/merge-patch+json')
+  ) {
```

Comment:
this question might be very obvious... is it still appropriate to include this logic in a function that's called `isContentTypeApplicationJson`? 🤔 or should we:
- have a separate function (`isContentTypeMergePatchJson`?) and check both of them in `backend_srv`?
- rename the function to be less specific?
- leave it as-is?

idk enough about the difference between `application/json` and `application/merge-patch+json` to have an opinion tbh 😅 just raising it

Reply:
@ashharrison90 - The context behind this content type is it's to work with the API platform. The Kubernetes API it's built off of has several options for doing patches, which are differentiated by the content type https://kubernetes.io/docs/reference/using-api/api-concepts/#patch-and-apply

The merge patch kind used here is the one that is most intuitive to most people when they think of patching an update - just a JSON object with the values you'd want to change. There will be no difference between `application/json` and `application/merge-patch+json` in what the content actually is - its all just JSON - but we need the specific content type for the patch to apply as needed.

I could go with separating this function, or renaming it to `isContentTypeJson` to be more generic. Whatever you prefer!

Reply:
i'd be fine with leaving the logic as-is and just renaming the function `isContentTypeJson` 👍 

but honestly i don't think my opinion should hold much weight, i'm sure you/app platform know best 🙌